routes/account.js

```diff
@@ -36,9 +36,17 @@ function accountRoutes (server, options, next) {
       var password = request.payload.data.attributes.password
       var id = request.payload.data.id
       var query = request.query
+
+      var currentTime = new Date().toISOString()
+      var createdAt = request.payload.data.attributes.createdAt
+      if (!createdAt) { // If createdAt is not assigned, use current time
+        createdAt = currentTime
+      }
```

Comment:
can you move `var createdAt = request.payload.data.attributes.createdAt` to the top of the function (bellow `var password = ...` and then replace `var createdAt = request.payload.data.attributes.createdAt` with `var createdAt = createdAt || currentTime`